tcms/core/ajax.py

```diff
@@ -167,31 +151,28 @@ def form(request):
     # Get the form
     q_app, q_form = q_app_form.split('.')[0], q_app_form.split('.')[1]
     exec('from tcms.%s.forms import %s as form' % (q_app, q_form))
-    try:
-        __import__('tcms.%s.forms' % q_app)
-    except ImportError:
-        raise
+    __import__('tcms.%s.forms' % q_app)
     q_app_module = sys.modules['tcms.%s.forms' % q_app]
     form_class = getattr(q_app_module, q_form)
-    form = form_class(initial=parameters)
+    form_params = form_class(initial=parameters)
 
     # Generate the HTML and reponse
-    html = getattr(form, 'as_' + q_format)
+    html = getattr(form_params, 'as_' + q_format)
     return HttpResponse(html())
 
 
-def tag(request, template_name="management/get_tag.html"):
+def tags(request, template_name="management/get_tag.html"):
     """Get tags for test plan or test case"""
 
     class Objects(object):
         __all__ = ['plan', 'case', 'run']
 
-        def __init__(self, request, template_name):
+        def __init__(self, req, template_name):
```

Comment:
Why change `request` to `req` ? `request` is much more readable and used in other places as well. It is better if we keep the longer descriptive name instead of an abbreviation.